### PR TITLE
Create the Selection Item List component

### PR DESCRIPTION
**Pull request by deenp03**
## Notion ticket link

[Create the Selection Item List component](https://www.notion.so/uwblueprintexecs/Create-the-Selection-Item-List-component-b98bd7a57c894186b0134cba12cdc78b)

## Implementation description
* Created the selection item list selection component and implemented it on the program details page
* Replaced the select component with item list selection component on Court information page
* edited the CustomInput file to make it more abstract

## Steps to test
1.[localhost.](http://localhost:3000/intake)

## What should reviewers focus on?
* Making sure that format is correct
* Component made correctly and abstractively

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/intake/Select.tsx [hunk 1]
```diff
@@ -1,27 +1,96 @@
 /* eslint-disable react/jsx-props-no-spreading */
```
Comment (bowenzhu1) [inline frontend/src/components/intake/Select.tsx:2]: we should change this file name to match the exported component, CustomSelectField.tsx?

**Review by bowenzhu1 — CHANGES_REQUESTED** [review]
minor comments, good work! hopefully this was a decent learning experience too

File: frontend/src/components/intake/CourtInformationForm.tsx [hunk 2]
```diff
@@ -122,15 +122,15 @@ const CourtInformationForm = ({
...
-                  defaultValue=""
-                >
-                  <option value="">Select First Nation heritage status</option>
-                  <option>First Nation Registered</option>
-                  <option>Eligible for Registration</option>
-                  <option>Inuit</option>
-                  <option>Metis</option>
-                  <option>Unknown</option>
-                </CustomSelectField>
+                  placeholder="Select First Nation heritage status"
```
[the remark this answers is not in the document]
Comment (deenp03) [reply frontend/src/components/intake/CourtInformationForm.tsx:125]: what icon should I make it?

[the remark this answers is not in the document]
Comment (bowenzhu1) [reply frontend/src/components/intake/CourtInformationForm.tsx:125]: should be same as the other ones, `                  iconRight={<Icon as={ChevronDown} />}`

**Review by bowenzhu1 — APPROVED** [review]
nice work!

**Pull request merged**